jetty-http2/http2-hpack/src/main/java/org/eclipse/jetty/http2/hpack/Huffman.java

```diff
@@ -450,13 +462,31 @@ private static int octetsNeeded(final int[][] table, String s)
         {
             char c = s.charAt(i);
             if (c >= 128 || c < ' ')
-                throw new IllegalArgumentException();
+                return -1;
             needed += table[c][1];
         }
 
         return (needed + 7) / 8;
     }
 
+    private static int octetsNeeded(final int[][] table,byte[] b)
+    {
+        int needed = 0;
+        int len = b.length;
+        for (int i = 0; i < len; i++)
+        {
+            int c = 0xFF & b[i];
+            needed += table[c][1];
+        }
+        return (needed + 7) / 8;
+    }
+
+    /**
+     * @param table The table to encode by
+     * @param buffer The buffer to encode to
+     * @param s The string to encode
+     * @return True if the string could be encoded, false otherwise (and the buffer may have been modified).
```

Comment:
The `@return` is wrong here. Is it a copy/paste error? I ask because the description seems accurate, but the method returns void so need to be double checked if it was indeed meaning to return a `boolean`.